### PR TITLE
apps/testcase/media: Add testcase for new api

### PR DESCRIPTION
**Pull request by YiHyunJin**
getAudioType_p, setAudioType_p and setAudioType_n are added.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [78053f6bdfe328cfb5a1e88024f65f50e89aead1] - Code Rule Check Result:
framework/include/media/DataSource.h:78: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/DataSource.h:78: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)

**Comment by seinfra** [comment]
Target : [78053f6bdfe328cfb5a1e88024f65f50e89aead1] - Code Rule Check (C++) OK.

File: framework/src/media/DataSource.cpp [hunk 1]
```diff
@@ -85,9 +85,14 @@ void DataSource::setPcmFormat(audio_format_type_t pcmFormat)
 	mPcmFormat = pcmFormat;
 }
 
-void DataSource::setAudioType(audio_type_t audioType)
+bool DataSource::setAudioType(audio_type_t audioType)
 {
+	if (audioType < AUDIO_TYPE_INVALID || AUDIO_TYPE_NOT_SUPPORTED <= audioType) {
```
Comment (Taejun-Kwon) [inline framework/src/media/DataSource.cpp]: I prefer below style.
if ((a < DEFINED_VALUE) || (a >= DEFINED_VALUE2))

**Comment by seinfra** [comment]
Target : [896d8e4b4c7d16fc9ab42b62208fee5412aca03d] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [896d8e4b4c7d16fc9ab42b62208fee5412aca03d] - Code Rule Check Result:
framework/include/media/DataSource.h:78: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/DataSource.h:78: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)

File: framework/src/media/DataSource.cpp (hunk 1 above)
Comment (YiHyunJin) [reply framework/src/media/DataSource.cpp]: @Taejun-Kwon  I will fix.

File: framework/include/media/MediaTypes.h [hunk 2]
```diff
@@ -103,7 +103,9 @@ typedef enum audio_type_e {
 	/** Audio type is FLAC */
 	AUDIO_TYPE_FLAC = 5,
 	/** Audio type is WAVE */
-	AUDIO_TYPE_WAVE = 6
+	AUDIO_TYPE_WAVE = 6,
+	/** Audio type is not supported */
+	AUDIO_TYPE_NOT_SUPPORTED
```
[the remark this answers is not in the document]
Comment (YiHyunJin) [reply framework/include/media/MediaTypes.h]: @Taejun-Kwon 
I want to set the standard when I set an unsupported audio type.

if (audioType < AUDIO_TYPE_INVALID || AUDIO_TYPE_NOT_SUPPORTED <= audioType) {
		meddbg("Not Supported audio type : %d\n", audioType);
		return false;
}

**Comment by seinfra** [comment]
Target : [cb69d2b15f15897b41a189938705e7b3d1510aa3] - Code Rule Check Result:
framework/include/media/DataSource.h:78: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/DataSource.h:78: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)

**Comment by seinfra** [comment]
Target : [cb69d2b15f15897b41a189938705e7b3d1510aa3] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [afaed952befc6fbfbe0128bad45a10663d6f27cd] -

**Comment by seinfra** [comment]
Target : [73c4d010efc76b47e6e62682b53646aef30aea4a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [73c4d010efc76b47e6e62682b53646aef30aea4a] - Code Rule Check OK.

**Pull request merged**